### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sonia-IDSA**
added Testbed implementations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ Used versions of the IDS-testbed components:
 - Dataspace Connector (DSC) v7.1.0
 - MetadataBroker (MDB) v5.0.0
 
+## IDS Reference Testbed implementations
+| Provider      | Description     |   Components used |  Public or Private Use? |
+| ------------- | --------------- |   --------------- | ------------------------
+| ...           | ...             |   ...    | .... |
```
Comment (IgorBalog-Eng) [inline README.md:44]: | Engineering | Verifying TRUEConnector compatibility with reference implementation | DSC v7, MDB v5, Omejdn v1.6 | Private |

**Comment by jfernandezsqs** [comment]
I think it is interesting to have this section `IDS Reference Testbed implementations` in the `README.md` file of the testbed. 
I suggest including a brief description at the beginning of the section. It could look something like this:

```
Please include in this pull request (link to the dedicated pull request) the successful IDS Reference Testbed implementations you have made. Including:
-	Name of your connector 
-	A brief description of it
-	Version of IDS-testbed components used (i.e. DSC v7.1.0, MDB v5.0.0, DAPS v1.6.0)
-	Whether it is public or private
```

We could have a dedicated pull request for this topic, where it is indicated that the developers can detail their implementations in that pull request to later be added to the IDS-testbed README file.
What do you think about this idea?

File: README.md (hunk 1 above)
Comment (jfernandezsqs) [reply README.md:44]: @IgorBalog-Eng thanks for the input.
Can you confirm the exact versions of the components used?
- DSC 7.1.0?
- MDB v5.0.0?
- DAPS v1.6.0?

Comment (IgorBalog-Eng) [reply README.md:44]: Yes, I confirm that those are the versions from docker-compose and .env file

**Review by jfernandezsqs — APPROVED** [review]
Checked, changes are correct

File: README.md (hunk 1 above)
Comment (jfernandezsqs) [reply README.md]: Perfect, @Sonia-IDSA has just submmited the changes to the [README.md](https://github.com/International-Data-Spaces-Association/IDS-testbed/tree/Sonia-IDSA-patch-1#readme) file.
@IgorBalog-Eng could you please review and approve this pull request?
Thanks in advance

**Review by IgorBalog-Eng — APPROVED** [review]
Changes related with Engineering TRUEConector are OK.

**Pull request merged**